pkg/reconciler/kubernetes/tektonpipeline/tektonpipeline.go

```diff
@@ -331,6 +339,29 @@ func makeInstallerSet(tp *v1alpha1.TektonPipeline, manifest mf.Manifest, tpSpecH
 	}
 }
 
+func (r *Reconciler) ensureTektonConfigExists(ctx context.Context, tp *v1alpha1.TektonPipeline) error {
```

Comment:
rename this function to something like (`targetNamespaceCheck` )
as our intention is to so 2 things:
1. if targetNamespace exists add extra labels (and remove it from manifest list)
2. keep it in manifest.


so now i think we can completey skip checking the status of TektonConfig as we are only interested in the namespace.

so let us use a `get` api call to check if the TektonPipeline spec.targetNamespace exists.
if yes then apply new labels then filter out namespaces from manifest
if no then do nothing and continue.

note: it is a good practice to use names to communicate the purpose of a function or variable